OpenSim/Simulation/Model/GeometryPath.h

```diff
@@ -85,7 +87,9 @@ OpenSim_DECLARE_CONCRETE_OBJECT(GeometryPath, ModelComponent);
     //--------------------------------------------------------------------------
 public:
     GeometryPath();
-    virtual ~GeometryPath();
+#ifndef SWIG
+    ~GeometryPath() override = default;
```

Comment:
The [SWIG manual](http://binf.gmu.edu/software/SWIG/SWIGPlus.html#SWIGPlus_nn7) says:

> If a C++ class does not define any public constructors or destructors, SWIG will automatically create a default constructor or destructor.

So I think we're fine here.